CONTRIBUTING.md

```diff
@@ -32,15 +32,18 @@ For example, if you wanted to change `cf-buttons`, here's what you'd do:
 1. `git clone git@github.com:cfpb/capital-framework.git`
 1. `cd capital-framework`
 1. `npm install`
-1. `git checkout -b doing-something-with-buttons canary`
+1. `git checkout -b button-fix canary`
 1. Edit file(s) in `/src/cf-buttons/` however you want.
 1. `npm run build` This will build every component (compiling Less, bundling JS, processing markdown docs) to `tmp/` in the project's root.
 1. Inspect and test your modified `cf-buttons` component in `tmp/`. See _Testing components locally_ below.
-1. Describe the changes in the "Unreleased" section of `/CHANGELOG.md`.
-Prepend the component's name to the described changes.
-1. `git commit -am "Describe your commit"`
-1. `git push upstream doing-something-with-buttons`
-1. Go to https://github.com/cfpb/capital-framework and open a pull request to merge `doing-something-with-buttons` into `canary`.
+1. When you're done hacking, open `/CHANGELOG.md` and add a line item describing your changes in the "Unreleased" section.
+Use the format: `- **cf-component-name:** [MAJOR|MINOR|PATCH] Description of change`.
+[Here's an example](https://github.com/cfpb/capital-framework/pull/291/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR12).
+  - If the change affects *all* components, put `**all components:**` as the name.
+  - If the change affects CF and not any components (e.g. if you're just updating [README.md](README.md)), put `**capital framework**`.
+1. `git commit -am "Fix button border radius"`
+1. `git push upstream button-fix`
```

Comment:
Should be `push origin` if this was cloned from the CFPB source, as suggested in step 1 of this process.